pkgs/desktops/gnome-2/desktop/gopenvpn/default.nix

```diff
@@ -20,12 +18,23 @@ in stdenv.mkDerivation rec {
   buildInputs = [ glib gtk2 dbus_glib gmime libnotify libgnome_keyring openssl cyrus_sasl gnonlin sylpheed gob2
                   gettext intltool libglade polkit pkgconfig autoconf automake expect openvpn makeWrapper ];
 
+  # disable the format hardening flags in order for the program to compile even though a potential vulnerability
+  # might be present in the form of a format string vulnerability. By failing to add this the program compilation
+  # will end with the following line:
+  #
+  #  > gopenvpn.c:978:10: error: format not a string literal and no format arguments [-Werror=format-security]
+  #
+  hardeningDisable = [ "format" ];
+
   patchPhase = ''
     # configure and remote extra "po/Makefile.in" in AC_CONFIG_FILES
     sed -i -e 's:AC_CONFIG_FILES(\[pixmaps/Makefile po/Makefile.in:AC_CONFIG_FILES(\[pixmaps/Makefile:g' configure.ac
     cat configure.ac | grep CONFIG_FILES
   '';
 
+  # configure with pkexec to be able to run openvpn as privileged user
+  configureFlags = ["--with-pkexec=${pkexecPath}/bin/pkexec"];
```

Comment:
I mixed `pkexecPath` up with a package variable. 
It should be:

```
configureFlags = ["--with-pkexec=${pkexecPath}/bin/pkexec"];
```

instead.